lib/error_reporter.h

```diff
@@ -218,6 +218,8 @@ class ErrorReporter {
         Util::SourcePosition position = sources->getCurrentPosition();
         position--;
 
+        // Unfortunately, we cannot go with statically checked format string
+        // here as it would require some changes to yyerror
```

Comment:
What are the changes?

Reply:
Ah, well, there is some logic there, but essentially the format string is chosen programmatically. And we need it constexpr here.